tutoraspects/templates/openedx-assets/assets/dashboards/Instructor_dashboard.yaml

```diff
@@ -89,7 +89,7 @@ metadata:
     targets:
     - column:
         name: org
-      datasetUuid: 4b274428-d781-41be-b362-ed6917443678
+      datasetUuid: 41278a97-d0ff-4645-9514-d79f80d275df
```

Comment:
Can you revert these uuids to their old values here and in the chart? I think changing them will cause duplication on existing installs.

Reply:
This is an intentional change to deprecate `dim_courses` in favor of using `course_names` directly. Since this is referring to the new physical dataset representing `course_names`, I would think this should apply cleanly to an existing installation. I thought I made my changes to the canonical instructor dashboard, so I don't think this is a case of importing a different version. Regardless, I'll try reverting my old environment and applying the changes in this branch to test the compatibility.

Reply:
Oh I see, I misread it. No worries.